### DISCUSSION TITLE
Lora usage makes VMRAM usage jump by several GBs? -Amateur Photography

### DISCUSSION THREAD

**Discussion by mald55** [lllyasviel/stable-diffusion-webui-forge#1452] [Q&A] [answered]
Hi There,

I am using flux1-dev-fp8 which I have set to 20 steps, and 960x1280, and I see the Vram usage cap at around 12GB give or take, but when using Loras (which are under 500 mb) the Vram usage jump drastically, to a point the generation basically stops. Is this normal behavior or a bug? 

I have a 4070ti super and the lora in question is https://civitai.com/models/652699/amateur-photography-flux-dev?modelVersionId=756149

Any ideas?

**Comment by lllyasviel** [comment, marked as answer]
update and try again

**Reply by mald55** [reply]
seems to be better now, I will keep testing. 

edit: actually the only thing that helps is setting the 'diffusion in low bits' to 'Automatic FP16 Lora'
